app/src/main/java/com/example/wnews/views/home/news/NewsView.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.wnews.views.home.news
+
+import com.example.wnews.models.News
+
+interface NewsView {
+
+    fun changeData()
```

Comment:
I'm assuming that this method is called when there is new data loaded from the presenter. In that case, this kind of event is usually described as an `onEventHappend`. For example:

`onClickListener`, `onActivityCreated`, `onScrollListener`, etc.

In this case, that we are receiving new data from the backend, I'd recommend describing this a `onNewsReceived` or `onNewPageReceived`.